src/exam/ExamAPIError.jsx

```diff
@@ -1,38 +1,66 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { useState, useContext } from 'react';
 import { Alert, Icon } from '@edx/paragon';
 import { Info } from '@edx/paragon/icons';
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import ExamStateContext from '../context';
 
-// Fixme: Replace with store state data once exam settings are implemented
-const platformName = 'edX';
-const contactUs = 'https://courses.edx.org/support/contact_us';
+export default function ExamAPIError() {
+  const state = useContext(ExamStateContext);
+  const { apiErrorMsg, proctoringSettings } = state;
+  const { contact_us: contactUs, platform_name: platformName } = proctoringSettings;
+  const [showError, setShowError] = useState(false);
 
-export default function ExamAPIError({ details }) {
-  return (
-    <Alert variant="danger">
-      <Icon src={Info} className="alert-icon" />
-      <Alert.Heading>
+  const renderHeading = () => {
+    if (contactUs && platformName) {
+      return (
         <FormattedMessage
-          id="exam.apiError.text"
+          id="exam.apiError.text1"
           defaultMessage={
             'A system error has occurred with your exam. '
-              + 'Please reach out to {support_link} for assistance, '
+              + 'Please reach out to {supportLink} for assistance, '
               + 'and return to the exam once you receive further instructions.'
           }
-          values={{ support_link: <a href={contactUs} target="_blank" rel="noopener noreferrer">{platformName} Support</a> }}
+          values={{ supportLink: <a href={contactUs} target="_blank" rel="noopener noreferrer">{platformName} Support</a> }}
         />
+      );
+    }
+
+    return (
```

Comment:
do we have tests for this error component?

Reply:
no, we don't
added them in 906b579